### PR TITLE
Added a customFilePath properties

### PR DESCRIPTION
**Pull request by Angelinsky7**
Hi,
i've not understand how to use this plugin when i use sass.
for that i've added a customFilePath that permits to change the path of each file...

like that i can do that

````
gulp.task('sass', function () {
    return gulp.src('./src/**/*.scss')
        .pipe(plugins.sourcemaps.init())
        .pipe(plugins.sass({
            /*importer: moduleImporter(),*/
            includePaths: [
                'src/',
                'node_modules/'
            ]
        }).on('error', plugins.sass.logError))
        .pipe(plugins.autoprefixer(autoprefixerSettings))
        .pipe(plugins.sourcemaps.write('.', { sourceRoot: '/../src' }))
        .pipe(gulp.dest('./public'))
        .pipe(browserSync.stream());
});

gulp.task('build:ts', ['sass'], function () {
    var tsResult = tsProject.src()
    .pipe(plugins.inlineNg2Template({
        base: '/src',
        useRelativePaths: true,
        removeLineBreaks: true,
        customFilePath: function (path) {
            if (path.indexOf('.css', path.length - '.css'.length) !== -1) {
                return path.replace('/src/', '/public/');
            }
            return path;
        }
    })).pipe(plugins.typescript(tsProject));

    //return tsResult.js.pipe(gulp.dest('./build'));

    return merge([
        tsResult.js.pipe(gulp.dest('./build')),
        tsResult.dts.pipe(gulp.dest('./build'))
    ]);
});
````

without inlineNg2Template complaining about not finding the path of the css file into the src folder

### CODE REVIEW

**Comment by ludohenin** [comment]
@Angelinsky7 Thanks for the PR!
I'll have a look shortly.

**Comment by murraybauer** [comment]
This would also be great for ionic project which use scss styles when publishing a web version

**Comment by Angelinsky7** [comment]
with pleasure !!!
i've made very simple but i think it would be easy and better to refactor it and adapt it more like your style...
maybe add a second parameter direclty with the file extension... or more properties

File: parser.js [hunk 1]
```diff
@@ -21,13 +21,18 @@ var defaults = {
...
   styleProcessor: defaultProcessor,
+  customFilePath: defaultSytlePath,
   supportNonExistentFiles: false
 };
 
 function defaultProcessor(path, file) {
   return file;
 }
 
+function defaultSytlePath(path) {
```
Comment (ludohenin) [inline parser.js]: Would you change the function name to `defaultCustomFilePath` ?

**Comment by ludohenin** [comment]
I've added a comment.
Would you also update the doc ?

**Comment by ludohenin** [comment]
@Angelinsky7 any updates ?

**Comment by Angelinsky7** [comment]
Sorry i was not there for the past weeks.
But i'm back... i agree with you. i'll update my fork tomorrow and let you know

**Comment by ludohenin** [comment]
@Angelinsky7 any updates ?

**Comment by Angelinsky7** [comment]
Is this ok for you ?
Sorry for the delay

**Comment by ludohenin** [comment]
@Angelinsky7 oops, would you merge from master, there are some conflicts here ?

**Comment by Angelinsky7** [comment]
OK... this time i think i'm good,
Sorry for all this mess, not very good at git

**Comment by ludohenin** [comment]
That's fine, thanks a lot!

**Pull request merged**